lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -535,13 +543,23 @@ def estimate_diffusion(
         }
 
         if method == "cve":
-            # We hardcode the blur constant for confocal for now (no motion blur)
+            try:
+                blur = self._kymo.motion_blur_constant
+            except NotImplementedError:
+                warnings.warn(
+                    RuntimeWarning(
+                        "Motion blur cannot be taken into account for this type of Kymo. As a "
+                        "consequence, not all estimates will be available."
+                    )
+                )
```

Comment:
"not all estimates" is pretty vague. it only affects the localization uncertainty estimates, not the diffusion estimates correct?

Reply:
Depends. If you pass in no localization variance, it affects the `stderr` for the diffusion constant and the `localization_variance`.  However, if you've passed in a `localization_variance`, it affects the diffusion constant itself as well.

Reply:
I have specified it more precisely now, and raise in the case where the diffusion constant is also unavailable (see last commit, which I kept separate).